pkg/gateway/nginx.go

```diff
@@ -143,12 +151,18 @@ func renderConfig(cfg *NGINXConfig, sm *ServiceMap) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func newLoggingNGINXManager() NGINXManager {
-	return &loggingNGINXManager{status: nginxStatusStopped}
+func newLoggingNGINXManager(hp int) NGINXManager {
+	manager := loggingNGINXManager{
+		status: nginxStatusStopped,
+		cfg:    newNGINXConfig(hp),
+	}
+	log.Printf("created newLoggingNGINXManager: %+v", manager)
+	return &manager
 }
 
 type loggingNGINXManager struct {
 	status string
+	cfg    NGINXConfig
```

Comment:
Let's log this before we instantiate a specific NGINXManager so we get this visibility regardless of the NGINXManager type.